### DISCUSSION TITLE
OBS Studio And "PrintScreen"

### DISCUSSION THREAD

**Discussion by GabrielRCL** [obsproject/obs-studio#4320] [Development Questions]
a curiosity, I play a certain game that I can't take screenshots of my screen. but I can do a live using obs studio and capture the OBS Studio screen.
why can't I capture the screen with my "PrintScreen" button on the keyboard, but with obs studio I can?

**Comment by WizardCM** [comment]
Two potential reasons:

1. The game is using exclusive fullscreen, bypassing the renderer that Snip & Sketch can capture
2. The game & OBS are running as admin, whereas Snip & Sketch isn't
